### ISSUE TITLE
Errors and bad transcript

### ISSUE THREAD

**Issue by gregseitz360** [purdy/aws-transcribe-transcript#9]
Thanks for this excellent script! I've used it several times and gotten exactly what I needed.

But recently, when executing the script via Terminal (I'm on a Mac), I get the following errors, and then the transcript file is created, but it's basically exactly the same as the input json.

Any help would be appreciated! 

PHP Warning:  file_get_contents(arsOutput.json): Failed to open stream: No such file or directory in /Applications/aws-transcribe-transcript-master/transcript.php on line 4

Warning: file_get_contents(arsOutput.json): Failed to open stream: No such file or directory in /Applications/aws-transcribe-transcript-master/transcript.php on line 4
PHP Warning:  Attempt to read property "results" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 8

Warning: Attempt to read property "results" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 8
PHP Warning:  Attempt to read property "speaker_labels" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 8

Warning: Attempt to read property "speaker_labels" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 8
PHP Warning:  Attempt to read property "segments" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 8

Warning: Attempt to read property "segments" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 8
PHP Warning:  foreach() argument must be of type array|object, null given in /Applications/aws-transcribe-transcript-master/transcript.php on line 10

Warning: foreach() argument must be of type array|object, null given in /Applications/aws-transcribe-transcript-master/transcript.php on line 10
PHP Warning:  Attempt to read property "results" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 17

Warning: Attempt to read property "results" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 17
PHP Warning:  Attempt to read property "items" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 17

Warning: Attempt to read property "items" on null in /Applications/aws-transcribe-transcript-master/transcript.php on line 17
PHP Warning:  foreach() argument must be of type array|object, null given in /Applications/aws-transcribe-transcript-master/transcript.php on line 22

Warning: foreach() argument must be of type array|object, null given in /Applications/aws-transcribe-transcript-master/transcript.php on line 22

**Comment by purdy** [comment]
Thanks!

The error you're getting is because the script cannot find/open the json file from AWS. It looks like a typo where you're using arsOutput.json vs asrOutput.json

**Comment by gregseitz360** [comment]
> Thanks!
> 
> The error you're getting is because the script cannot find/open the json file from AWS. It looks like a typo where you're using arsOutput.json vs asrOutput.json

Thanks so much for your prompt response! Unfortunately, that's not it. I've checked and rechecked the file names and commands for typos. I've even tried renaming the JSON files to simple filenames and using those instead. It always has the same result. I just re-downloaded the AWS JSON file, copied the file name, and pasted it into the Terminal command. That said, I don't doubt it's my mistake, just can't pinpoint the problem. Thanks again.

EDIT: After re-running the AWS transcription job and downloading the JSON file, the script seems to have worked. I did set up speaker partitioning this time, which I believe was the only difference. I've run into this problem intermittently in the past few months, but I'm good to go for now and will file a new ticket if I encounter the issue again. Thanks again.
